tsdb/head.go

```diff
@@ -559,6 +559,8 @@ func newHeadMetrics(h *Head, r prometheus.Registerer) *headMetrics {
 
 func mmappedChunksDir(dir string) string { return filepath.Join(dir, "chunks_head") }
 
+func MmappedChunksDir(dir string) string { return mmappedChunksDir(dir) }
```

Comment:
You could probably just export the method above? I imagine you've done this to avoid changing all the calls throughout the codebase?

Reply:
I can just use the private one as we're in the same package.
(this was just a PoC, a lot of stuff are TBD)